tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,94 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-6-0-BYOS.x86_64-1.0.0-EC2-Build1.36.raw.xz
+#  - Install the required dependencies to install the aistack helm chart and containers
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use transactional;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+
+sub install_dep_pkg {
+    my ($instance, $package) = @_;
+
+    record_info("Install $package",
```

Comment:
You could add here also a check if already installed like `rpm -q ..`, like that: `return if (...rpm -q pkg == 0)`

Reply:
Also, you could add, before here, a check if already pkg installed, like that:
`return if (...rpm -q pkg == 0)`  
or check system status and act accordingly.

Reply:
Fixed